crates/bounded_planet_core/src/bin/client/main.rs

```diff
@@ -1,11 +1,11 @@
-use std::{fs, net::ToSocketAddrs, path::PathBuf, time::Duration};
+use std::{fs, net::ToSocketAddrs, path::PathBuf, sync::Arc, time::Duration};
 
 use structopt::StructOpt;
 use bevy::prelude::*;
-use bevy::{app::ScheduleRunnerPlugin};
+use bevy::app::ScheduleRunnerPlugin;
 
-use bounded_planet::networking::components::Connection;
-use tracing::{Level, info, warn};
+use bounded_planet::networking::{events::{ReceiveEvent, SendEvent}, packets::{Packet, Ping, Pong, StreamType}, systems::NetEventLoggerState, systems::log_net_events};
```

Comment:
Gotta unroll these imports!

First line is also a good candidate for unrolling.